Assets/MRTK/SDK/Experimental/Dialog/README_Dialog.md

```diff
@@ -1,5 +1,5 @@
 # Dialog
-![](/Documentation/Images/Dialog/MRTK_UX_Dialog_Main.png)
+![Dialog](../../../../../Documentation/Images/Dialog/MRTK_UX_Dialog_Main.png)
```

Comment:
Should I enable docs checking under the Asset path as well? I added tools that would detect invalid image paths, and it seems like this would have really come in handy here.

@keveleigh @CDiaz-MS thoughts?

Reply:
Yeah seems reasonable to me!

Reply:
Done!